changelogs/unreleased/121-pawanpraka1

```diff
@@ -0,0 +1 @@
+Fixes an issue where PVC was bound to PV because of invalid/wrong parameters in PVC/Storageclass.
```

Comment:
n_ : Fixes an issue where PVC was bound to invalid PV created using incorrect values provided in PVC/Storageclass. 

Need to find a better word for "invalid" to qualify this PV.

Reply:
made it unusable.
Fixes an issue where PVC was bound to unusable PV created using incorrect values provided in PVC/Storageclass.